services/scabbard/libscabbard/src/service/v3/timer_filter.rs

```diff
@@ -174,7 +174,10 @@ mod tests {
             .with_alarm(SystemTime::now())
             .with_coordinator(fqsi.clone().service_id())
             .with_epoch(1)
-            .with_participant_processes(vec![peer_service1.clone()])
+            .with_participants(vec![Participant {
```

Comment:
Type in commit message: ParticiantContext -> ParticipantContext

Reply:
Fixed